egs/wsj/s5/steps/cleanup/append_eval_to_ctm.py

```diff
@@ -0,0 +1,396 @@
+#!/usr/bin/env python
+
+# Copyright 2016  Vimal Manohar
+# Apache 2.0
+
+from __future__ import print_function
+import sys, operator, argparse
+
+# Append to the CTM the Levenshtein alignment of 'hypothesis' and 'reference':
+# (i.e. the output of 'align-text' post-processed by 'wer_per_utt_details.pl')
+
+# The tags in the processed Levenshtein alignment are:
```

Comment:
Please give an example of what a CTM line looks like before you do this, and what it looks like after you do this.
Normally the word 'tag' implies that it's optional, and zero or more tags is allowed.  You might want to use the word 'token' or 'label' if exactly one per line will be present.